soccer/NewRefereeModule.cpp

```diff
@@ -328,6 +334,18 @@ void NewRefereeModule::updateGameState(bool blueTeam) {
             break;
         case Command::GOAL_BLUE:
             break;
+        case Command::BALL_PLACEMENT_YELLOW:
```

Comment:
Should we add `_state.gameState.state = GameState::Stop;` for this and the blue placement to make sure we're in the correct state?